lib/compat/wordpress-6.8/post.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Temporary compatibility shims for block APIs present in Gutenberg.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Get the available rendering modes for the Block Editor.
+ *
+ * post-only: This mode extracts the post blocks from the template and renders only those.
+ * The idea is to allow the user to edit the post/page in isolation without the wrapping template.
+ *
+ * template-locked: This mode renders both the template and the post blocks
+ * but the template blocks are locked and can't be edited. The post blocks are editable.
+ *
+ * @return array Array of available rendering modes.
+ */
+function gutenberg_post_type_rendering_modes() {
+	return array(
+		'post-only',
+		'template-lock',
```

Comment:
Why is there a "template-lock" rendering mode, AFAIK, there's only two modes.

Reply:
I'm not finding any other references of this in the codebase. Removed 👍